### PR TITLE
Addition of Policy and Caution to Porting Guide

### PR DESCRIPTION
**Pull request by samirsahu1947**
Addition of Policy and Caution to Porting Guide

### CODE REVIEW

**Comment by seinfra** [comment]
Code Rule Check OK.

File: docs/Porting_Guide.md [hunk 1]
```diff
@@ -1,16 +1,29 @@
...
 > [Network](#network)  
 > [Audio](#audio)  
 > [Confirmation of Porting](#confirmation-of-porting)
 
-## Policy & Causion
-1. Use driver, framework and system call APIs.  
+## Policy & Caution
+1. Driver Implementation Recommendation
+Instead of hooking the default driver interface code (which is a part of SoC package) to the platform,
+it is advisable to include the drivers accourding the TizenRT platform driver model.
```
Comment (lokeshbv) [inline docs/Porting_Guide.md]: Please check typo `accourding`

File: docs/Porting_Guide.md [hunk 2]
```diff
@@ -1,16 +1,28 @@
...
+
+2. New Application Inclusion
+It is a good practise to add the Kconfig entry. All application configurations and dependencies should be included properly.
+
+3. Use driver, framework and system call APIs.
 Don't call architecture codes from application directly. When protected build enables, it will be blocked.
+Application must call TizenRT driver interface to talk to underlying driver.
+
+4. Maintain TizenRT Code Structure
+It is not advised if addition of peripherals, new interfaces, new open source code etc are done by aligning to default board package structure.  Understanding the TizenRT code structure and adhering to the structure helps in better architecture desgin and portability.
```
Comment (sunghan-chang) [inline docs/Porting_Guide.md]: Remove a space. (There are two spaces)
```
structure.  Understanding
```

File: docs/Porting_Guide.md [hunk 3]
```diff
@@ -1,16 +1,28 @@
...
 > [Network](#network)  
 > [Audio](#audio)  
 > [Confirmation of Porting](#confirmation-of-porting)
 
-## Policy & Causion
-1. Use driver, framework and system call APIs.  
+## Policy & Caution
+1. Driver Implementation Recommendation
+Instead of hooking the default driver interface code (which is a part of SoC package) to the platform,
+it is advisable to include the drivers according the TizenRT platform driver model.
```
Comment (sunghan-chang) [inline docs/Porting_Guide.md]: You didn't add two spaces at end of previous line. What is intention? Do you want to write them in same line or different lines?

**Comment by sunghan-chang** [comment]
@samirsahu1947 Could you see yours at markdown view?
https://github.com/samirsahu1947/TizenRT/blob/porting_guide_policies/docs/Porting_Guide.md#policy--caution

**Comment by seinfra** [comment]
Code Rule Check OK.

**Comment by seinfra** [comment]
Code Rule Check OK.

**Comment by seinfra** [comment]
Code Rule Check OK.

File: docs/Porting_Guide.md [hunk 4]
```diff
@@ -1,16 +1,29 @@
 # Porting Guide
 
 ## Contents
-> [Policy & Causion](policy--causion)  
+> [Policy & Caution](policy--caution)
```
Comment (lokeshbv) [inline docs/Porting_Guide.md]: link to `Policy & Caution `is missing. I think this line should be changed to `[Policy & Caution](#policy--caution)`

**Pull request merged**